<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-5GDB5GM" failures="1" tests="13" name="Full Regression Test" time="354.819" errors="0" timestamp="2025-06-10T16:37:07 EDT">
  <testcase classname="tests.LoginTest" name="loginTest" time="1.205"/>
  <testcase classname="tests.LoginTest" name="loginTest" time="0.84"/>
  <testcase classname="tests.LoginTest" name="loginTest" time="1.043"/>
  <testcase classname="tests.AddToCartTest" name="testAddToCart" time="18.289"/>
  <testcase classname="tests.CheckoutValidationTest" name="testEmptyCheckoutForm" time="20.302"/>
  <testcase classname="tests.CheckoutDataDrivenTest" name="checkoutFormTest" time="19.089"/>
  <testcase classname="tests.CheckoutDataDrivenTest" name="checkoutFormTest" time="18.838"/>
  <testcase classname="tests.CheckoutDataDrivenTest" name="checkoutFormTest" time="18.944"/>
  <testcase classname="tests.InvalidCheckoutFormTest" name="invalidCheckoutTest" time="19.158"/>
  <testcase classname="tests.InvalidCheckoutFormTest" name="invalidCheckoutTest" time="18.943"/>
  <testcase classname="tests.InvalidCheckoutFormTest" name="invalidCheckoutTest" time="18.562"/>
  <testcase classname="tests.InvalidCheckoutFormTest" name="invalidCheckoutTest" time="18.642"/>
  <testcase classname="tests.InvalidCheckoutFormTest" name="invalidCheckoutTest" time="18.616">
    <failure type="java.lang.AssertionError" message="Expected an error message but none was found did not expect [No error message found] but found [No error message found]">
      <![CDATA[java.lang.AssertionError: Expected an error message but none was found did not expect [No error message found] but found [No error message found]
at tests.InvalidCheckoutFormTest.invalidCheckoutTest(InvalidCheckoutFormTest.java:56)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- invalidCheckoutTest -->
</testsuite> <!-- Full Regression Test -->
